Feature spring boot starter template (#10)
harsh-2711/getting-started: backend/Spring Boot/springboot_starterproject/README.md

# Spring Boot Starter Project

## 📌 Introduction

> This Project contains basic template for Spring Boot.<br>

Reviewer: Some detailed intro can be useful here. Like apart from the template what else does it includes (eg. which API have you designed).

## 📌 What You will build using this project
You will build a simple web application with Spring Boot.<br>
There will be a health API to check project is working correctly.<br>

## 📌 What You will need
IntelliJ IDEA<br>
Java installed with version 8 or higher<br>
Maven<br>

## 📌 Ways to create Spring Boot Starter Project
Using Spring.io initializer<br>
Using IntelliJ or any similar IDE and Maven simple project<br>
Using Spring Tool Suite <br>
Using CLI<br>

> 

Reviewer: Extra lines. I guess you have added this everywhere, so I'm not commenting on it everywhere now. Please do this spacing change. Also, keep only a single blank line when starting new headers.

## 📌 Important points to mark while creating Spring Boot Application
Spring Boot creates ```Stane-ALone``` Application. It means you will not require to install Tomcat Sever on your own (Perks of using Spring Boot)<br>

> Default Port to run a application is 8080. We can change port of application in application.properties file. Check application.properties file to check port configration changed by me.

Reviewer: ```suggestion
Default Port to run an application is 8080. We can change the port of application in the `application.properties` file. Check the `application.properties` file to check port configuration changed by me.

```

> 

Reviewer: Extra lines not needed.
